scripts/templates/clusters.html

```diff
@@ -14,18 +14,65 @@
             text-align: right;
             white-space: pre;
         }
+
+        ol ::marker {
+            font-weight: bold;
+        }
     </style>
 {% endblock %}
 
 {% block content %}
-<h2>Clusters</h2>
-<p>Sorted by size</p>
-
-<ol>
-    {% for cluster in clusters %}
-    <li class="pl1 pb2"><div class="f3 lh-copy">{{ cluster.label }}</div>
-        {{ cluster.count }} Documents
-    </li>
-    {% endfor %}
-</ol>
+{# two columns #}
+<div class="mw9 center ph3-ns">
+  <div class="cf ph2-ns">
+    <div class="fl w-100 w-40-ns pa2">
+
+    <h2>Clusters</h2>
+    <p>Sorted by size</p>
+
+    <ol>
+        {% for cluster in clusters %}
+        <li class="pl1 pb2"><a class="dib no-underline dim black v-mid" href="?cluster={{ cluster.value }}">
+            <div class="f3 lh-copy {% if cluster.selected %}blue{% endif %}">{{ cluster.label }}</div>
+            {{ cluster.count }} Documents
+            </a>
+        </li>
+        {% endfor %}
+    </ol>
+    </div> {# end first column #}
+
+    {# second column, when there are documents #}
+    {% if documents %}
+    <div class="fl w-100 w-60-ns pa2">
+      <div class="bl-ns bt bt-0-ns pv4">
+        <div class="pl3">
+            <div class="f2 lh-copy blue">{{ current_cluster }}</div>
+            <div>{{ current_cluster_count }} Documents</div>
```

Comment:
tiny thing, but the design says "X total documents", which would matter more if we put counts on the (sub?)clusters as well

Reply:
Oh, I missed that. Interesting point about counts on the sub clusters, that seems like it could be useful.